### DISCUSSION TITLE
Google推出可以辨認手寫字

### DISCUSSION THREAD

**Discussion by delia07152** [1101-GS4719-C/Python-Community-GS4719-C#915] [General]
雖然我不清楚他實際是怎麼做到的，但我覺得有這項技術後，的確能減少藥劑師的負擔以及出錯的機率。不過我認為這個僅能作為輔助，如果真的不確定還是直接詢問醫師佳！

https://technews.tw/2022/12/21/google-lens-recognizes-doctor-handwriting/

**Comment by Kotori0501** [comment]
Google lens這個應用軟體真的很讚，我都用來查詢大學題目的解答，有時還真的查到了，真棒。

**Comment by dabbie109303501** [comment]
但醫生的手寫體連其他人都不一定能辨別出來，感覺要讓ai達到極高的辨認率蠻難的

**Comment by peggy0329** [comment]
這個新技術真不錯，或許未來不只可以用在醫生的手寫字上，也可以讓學校老師拿來辨識一些學生字寫很醜的作業XD 但蠻好奇背後的技術為何？會不會錯誤率很高呢？

**Comment by Evan0979** [comment]
自應該也是要夠好看,我寫的中文,系統辨認完可能會輸出英文哈哈哈哈哈

**Comment by ruthchen1020** [comment]
過去很多時候都靠google lens看原文書，也希望google可以先將辨識符號多做修正，公式透過鏡頭就變亂碼，d翻譯成丁，真的覺得很好笑...而且辨識筆跡是很早以前的技術吧，手機以前就有手寫輸入阿。

**Comment by evanchou1204** [comment]
既然想要看懂醫生的鬼畫符，為什麼不讓醫生直接用打字的就好了。

**Reply by StevenWanglolz** [reply]
你問的很有道理，不愧是邏輯鬼才

**Comment by StevenWanglolz** [comment]
或者是統一用鍵盤打字?效率更高，而且全面電子化也比較方便吧

**Comment by CPCpyt** [comment]
我已經能夠想到在未來寫在紙本上的手稿能直接轉換成數位資料的情況了。
